torch/nn/quantized/modules/linear.py

```diff
@@ -37,7 +37,9 @@ def forward(self, X):
 
     @staticmethod
     def from_float(mod):
```

Comment:
Should we have this API instead be:
from_float(mod, weight_qparams, activation_qparams)
mod is a nn module: Without observers, qconfig
Benefits:  Allows for users to still use from_float and specify directly the qparams without using our mechanism of observers etc

Reply:
OK